src/components/forms/fields/InputFieldWithTags/InputFieldWithTags.stories.tsx

```diff
@@ -49,14 +49,16 @@ export const InputWithTags: Story = {
     };
 
     return (
-      <Card>
+      <Card style={{width: '350px'}}>
         <InputFieldWithTags
           tags={tags}
           value={inputText}
+          label={'Input with tags'}
           onUpdate={handleUpdate}
           onUpdateTags={handleUpdateTags}
-          placeholder=""
+          placeholder="Placeholder"
         />
+        <p>(enter creates new tags; backspace removes tags)</p>
```

Comment:
Maybe better to put this in an info alert above the card, rather than inline. Also if you can make the above sentence case (so capital Enter and ending in a period, without the parens).